package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "odoc",
+  "version": "1.2.0",
+  "description": "The OCaml/Reason Documentation Generator",
+  "main": "index.js",
```

Comment:
Hm, are you trying to sneak in some uncommited JS? 😛

Reply:
I believe that will be a generated file.

Reply:
This was generated. Should be removed from the `package.json`!